dispatch/BESUtil.cc

```diff
@@ -1128,8 +1128,9 @@ void ios_state_msg(std::ios &ios_ref, std::stringstream &msg) {
  * Thanks to O'Reilly: https://www.oreilly.com/library/view/c-cookbook/0596007612/ch10s08.html
  * @param file_name
  * @param o_strm
+ * @return The number of bytes read/written
  */
-void BESUtil::file_to_stream(const std::string &file_name, std::ostream &o_strm)
+uint64_t BESUtil::file_to_stream(const std::string &file_name, std::ostream &o_strm, uint64_t read_start_position)
 {
     stringstream msg;
```

Comment:
This should be moved into something that won't be run when we're not debugging. It could all be moved inside the BESDEBUG() marco.

Reply:
This makes no sense. The line numbers are relevant to your comment.

Reply:
I sorted it out